drift.py

```diff
@@ -169,7 +106,11 @@ def main():
     if args.to_commit != None:
         to_commit = args.to_commit
 
-    da = DriftAnalysis([args.specs])
+    specs = None
+    if args.specs != None:
```

Comment:
I think you can just say if args.specs (and not do the exact comparison to None). I don't know if Todd likes this way better, but the version without None is considered more Pythonic.

Reply:
Won't we get an error if we try to call split on none though? Or is that okay since technically specs is a required input?

Reply:
The `if args.specs` won't be entered if args.specs is None, empty string, 0, or empty list. It's just writing it differently.

Reply:
Gotcha! This makes sense.